src/GetOpt.php

```diff
@@ -399,7 +399,7 @@ public function setHelpLang($language = 'en')
     /**
      * Translate $key
      *
-     * Returns the key if no translation is found
+     * Returns english fallback or the key if no translation is found
```

Comment:
Suggested rewording 

> Returns the translation for the given key; falls back to English if it is not localized in the configured language, and ultimately returns the key itself should it not exist in the English language file.

Reply:
It was just too much text for me. But your are right: that is what the function is doing :smile:

Reply:
Yeah, who cares about documentation anyway ? Just read the f***ing code, right ? :stuck_out_tongue_winking_eye:

Reply:
See #117